Geometry/MuonCommonData/plugins/dd4hep/DDMuonAngular.cc

```diff
@@ -3,54 +3,50 @@
 #include "DetectorDescription/DDCMS/interface/DDPlugins.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 
-using namespace std;
-using namespace dd4hep;
-using namespace cms;
 using namespace cms_units::operators;
 
-static long algorithm(Detector& /* description */,
+static long algorithm(dd4hep::Detector& /* description */,
                       cms::DDParsingContext& context,
                       xml_h element,
-                      SensitiveDetector& /* sens */) {
+                      dd4hep::SensitiveDetector& /* sens */) {
   cms::DDNamespace ns(context, element, true);
-  DDAlgoArguments args(context, element);
+  cms::DDAlgoArguments args(context, element);
 
   int n = args.value<int>("n");
   int startCopyNo = args.find("startCopyNo") ? args.value<int>("startCopyNo") : 1;
   int incrCopyNo = args.find("incrCopyNo") ? args.value<int>("incrCopyNo") : 1;
   float startAngle = args.value<float>("startAngle");
```

Comment:
@bsunanda , there are still few float.

Reply:
I do not think they do not matter - all the relevant operations are in double